flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/container/FlinkContainersBuilder.java

```diff
@@ -246,7 +246,7 @@ public FlinkContainers build() {
 
     private GenericContainer<?> buildZookeeperContainer() {
         return configureContainer(
-                new GenericContainer<>(DockerImageName.parse("zookeeper").withTag("3.4.14")),
+                new GenericContainer<>(DockerImageName.parse("zookeeper").withTag("3.5.9")),
```

Comment:
What about loading the value from the pom file instead of having it duplicated here? 🤔

Reply:
I created a ticket for that: https://issues.apache.org/jira/browse/FLINK-25886

We'll have to see how easy it is to set this up while still making things work out-of-the-box in IntelliJ.